exporter/kafkaexporter/authentication.go

```diff
@@ -68,6 +87,27 @@ func configurePlaintext(config PlainTextConfig, saramaConfig *sarama.Config) {
 	saramaConfig.Net.SASL.Password = config.Password
 }
 
+func configureSASL(config SASLConfig, saramaConfig *sarama.Config) error {
+	saramaConfig.Net.SASL.Enable = true
+	saramaConfig.Net.SASL.User = config.Username
```

Comment:
Is there a validation somewhere, ensuring that the username and password aren't empty?